source/js/nav-newsletter.js

```diff
@@ -132,7 +136,7 @@ class NavNewsletter {
       0}/`;
     props.csrfToken = props.csrfToken || csrfToken;
     props.isHidden = false;
-    this.form = ReactDOM.render(<JoinUs {...props} />, elements.joinUs);
+    this.form = ReactDOM.render(<I18nProvider language={getCurrentLanguage()} catalogs={catalogs}><JoinUs {...props} /></I18nProvider>, elements.joinUs);
```

Comment:
Note: this makes `this.form.reset()` fail, and I’m not sure how to work around it

Reply:
What if we move this into the main `render()` output of the `JoinUs` component instead?

Reply:
I’m still hitting that weird issue, unfortunately. Even if I move `<I18nProvider>` inside the `<form>` element.